### DISCUSSION TITLE
Ventura stuck or restart in AMD rayzen cpu?

### DISCUSSION THREAD

**Discussion by markaltmaier** [quickemu-project/quickemu#1274] [Quickemu]
![01](https://github.com/quickemu-project/quickemu/assets/72730763/19fa1835-6800-4c78-b234-b5de2df5a7f8)

It always stuck, and restart, I don't know why?

**Comment by omegaui** [comment]
@Mustafa-Agha Can you check the logs? I think its a kernel panic. I have faced this issue while installing macos on virtualbox.

**Reply by markaltmaier** [reply]
I think amd is not supported, so I changed my laptop for intel cpu. sorry I can't reproduce the steps again.

**Reply by nayeemtby** [reply]
> @Mustafa-Agha Can you check the logs? I think its a kernel panic. I have faced this issue while installing macos on virtualbox.

I kind of have the same issue on a ryzen 5600g with 1660 super. The logs stop at random points, I think based on how long it takes me to select `macOS Base System`. If I select late, then it stops at a earlier point in the logs, and if I select early, it stops at a later point in the logs. As if there's a time limit on the VM or something.

It stops at here if I select late
![image](https://github.com/user-attachments/assets/b9d68277-f13e-454c-9c10-bed1fa6076d6)


The points are being more random when I am trying with sonoma
![image](https://github.com/user-attachments/assets/dabb7bd0-4933-4cae-b1f4-4adbe29871de)

I tried ventura again while writing this and on the first time it went through the logs and brought me back to the UEFI screen. And the subsequent tries got stuck on the logs. And this one seems like the whole thing froze because the word `scheduled` does not get fully printed
![image](https://github.com/user-attachments/assets/85f9fa79-b572-4bd4-9ceb-aa4522985ba8)
